File: AppPrueba/app/src/main/java/com/example/felipehernandez/appprueba/MainActivity.java
package com.example.felipehernandez.appprueba;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    private Button btnClick;

    private int count =  1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


       btnClick  =  findViewById(R.id.btnClick);


       btnClick.setOnClickListener(new View.OnClickListener() {
           @Override
           public void onClick(View v) {
               btnClick.setText(String.format(" %s clicks!",(count++)));
           }
       });
    }


}

File: App4kotlin/app/src/main/java/felipehernandez/com/app4kotlin/MainActivity.kt
package felipehernandez.com.app4kotlin

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        val listaElementos = arrayOf("Sumar","Restar")

        val adaptador = ArrayAdapter<String>(this,
                android.R.layout.simple_spinner_item,
                listaElementos)

        spinnerOperaciones.adapter  = adaptador


        btnAceptar.setOnClickListener {

            var elementoSelecionado = spinnerOperaciones.selectedItem.toString()

            var resultado = 0

            if (elementoSelecionado.equals("Sumar")){

                 resultado  =  9 +  txtNumero.text.toString().toInt()
            }else {
                 resultado  =  9 -  txtNumero.text.toString().toInt()
            }

            var OperacionesExtras = ""

            if(chkMultiplicar.isChecked){
                OperacionesExtras += " El resultado de la operacion Multiplicar es ${9 * txtNumero.text.toString().toInt()}"
            }

            if(chkDividir.isChecked){
                OperacionesExtras += " El resultado de la operacion Dividir es ${9 / txtNumero.text.toString().toInt()}"
            }

            lblResultado.text = "El resultado de la operacion ${elementoSelecionado} ${resultado} ${OperacionesExtras}"

        }
    }
}
